fail2ban/test_fail2ban.py

```diff
@@ -0,0 +1,155 @@
+# stdlib
+from nose.plugins.attrib import attr
```

Comment:
The CI linting step is failing on this unused import, can you remove it? (see https://travis-ci.org/DataDog/integrations-extras/jobs/249699598#L1428)